DesignSafe-CI/portal
# DES-661 - Fix user-agent warning for supported IOS browsers.

## Overview: ##
Fixes an incorrect error message that pops up on supported browsers on IOS devices.

## PR Status: ##


## Related Jira tickets: ##

* [DES-661](https://jira.tacc.utexas.edu/browse/DES-661)

## Summary of Changes: ##
Added a separate condition for IOS user-agent names (FxiOS, CriOS) for supported browsers. 

## Testing Steps: ##
1. Open the site on supported browsers (Firefox and Chrome) on IOS.
2. Confirm that you do not see the error message about unsupported browsers.

## UI Photos:
N/A

## Notes: ##
I have tested with two IOS devices and my Android phone with both Firefox and Chrome on each device to confirm correct behavior.
